### PR TITLE
[terra-framework] Removed inline style for dev-site components Part 4

### LINKED ISSUE #657: Remove inline styles in dev site components
# Feature Request

## Description

We want to remove inline styles from our dev site components to set a better example for consumers of terra.  As a replacement, we should pass in classes and use css to style the components.

### PR DESCRIPTION
**Pull request by avinashg1994**
### Summary
Removed all inline styles from Dev-site components and replaced the same styles with external css for the **Popup** package.

Partially resolves #657 

Thanks for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
Can we align the styles in these examples and use a single stylesheet instead of adding a custom style sheet for each example?

File: packages/terra-popup/src/terra-dev-site/doc/example/PopupNoHeader.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:local {
```
Comment (ShettyAkarsh) [inline packages/terra-popup/src/terra-dev-site/doc/example/PopupNoHeader.scss]: This file is same code as popupbounded.scss. can we resuse them by renaming the file ?

**Comment by avinashg1994** [comment]
> Can we align the styles in these examples and use a single stylesheet instead of adding a custom style sheet for each example?

Yes, it is possible to have just a single file with all the styles. But due to the following reasons we decided with the current approach:
- For the purpose of modularity and readability.
- Multiple examples have elements, which serve the same purpose, have different styles, e.g. "content-wrappers".
- Having a common stylesheet will require unique class names for different elements, which will in-turn make them very long.

Even so, i have tried to combine styles from multiple files into a single stylesheet to reduce the number of redundant stylesheets where ever possible.
@emilyrohrbough, Please share your thoughts on this.

File: packages/terra-popup/src/terra-dev-site/doc/example/PopupNoHeader.scss (hunk 1 above)
Comment (avinashg1994) [reply packages/terra-popup/src/terra-dev-site/doc/example/PopupNoHeader.scss]: I agree, i have renamed and reused PopupBounded.scss and removed PopupNoHeader.scss.

**Pull request merged**